### PR TITLE
New version: GLPI-Project.GLPI-Agent version 1.11

### PR DESCRIPTION
**Pull request by spectopo**
Created by [🥟 Dumplings](https://github.com/SpecterShell/Dumplings) in workflow run [#8314](https://github.com/SpecterShell/Dumplings/actions/runs/11011581746).

<details>

<summary>Log</summary>

````
Updated: 1.10 → 1.11
Submitting WinGet manifests
Creating manifests
Uploading manifests and making commits
Creating a pull request
````

</details>

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/174923)

### CODE REVIEW

File: manifests/g/GLPI-Project/GLPI-Agent/1.11/GLPI-Project.GLPI-Agent.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+PackageIdentifier: GLPI-Project.GLPI-Agent
+PackageVersion: "1.11"
+PackageLocale: en-US
+Publisher: Teclib'
+PublisherUrl: https://glpi-project.org/
+PublisherSupportUrl: https://github.com/glpi-project/glpi-agent/issues
+PrivacyUrl: https://glpi-project.org/privacy-policy/
+Author: Teclib'
+PackageName: GLPI Agent
+PackageUrl: https://glpi-project.org/downloads/
```
Comment (g-bougard) [inline manifests/g/GLPI-Project/GLPI-Agent/1.11/GLPI-Project.GLPI-Agent.locale.en-US.yaml:13]: Package URL is wrong as it only provides server part downloads, not agent related one. Better use the github release page.

```suggestion
PackageUrl: https://github.com/glpi-project/glpi-agent/releases
```

File: manifests/g/GLPI-Project/GLPI-Agent/1.11/GLPI-Project.GLPI-Agent.locale.zh-CN.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+PackageIdentifier: GLPI-Project.GLPI-Agent
+PackageVersion: "1.11"
+PackageLocale: zh-CN
+Publisher: Teclib'
+PublisherUrl: https://glpi-project.org/
+PublisherSupportUrl: https://github.com/glpi-project/glpi-agent/issues
+PrivacyUrl: https://glpi-project.org/privacy-policy/
+Author: Teclib'
+PackageName: GLPI Agent
+PackageUrl: https://glpi-project.org/downloads/
```
Comment (g-bougard) [inline manifests/g/GLPI-Project/GLPI-Agent/1.11/GLPI-Project.GLPI-Agent.locale.zh-CN.yaml:13]: Package URL is wrong as it only provides server part downloads, not agent related one. Better use the github release page.

```suggestion
PackageUrl: https://github.com/glpi-project/glpi-agent/releases
```

File: manifests/g/GLPI-Project/GLPI-Agent/1.11/GLPI-Project.GLPI-Agent.locale.zh-CN.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+# Moniker:
+Tags:
+- glpi
+# ReleaseNotes:
+ReleaseNotesUrl: https://github.com/glpi-project/glpi-agent/blob/HEAD/Changes
+PurchaseUrl: https://glpi-project.org/prices/
+# InstallationNotes:
+Documentations:
+- DocumentLabel: 文档
+  DocumentUrl: https://glpi-project.org/documentation/
```
Comment (g-bougard) [inline manifests/g/GLPI-Project/GLPI-Agent/1.11/GLPI-Project.GLPI-Agent.locale.zh-CN.yaml:34]: Documentation URL is wrong as it's the server part one, not agent related one. Better use the RTD documentation url.
```suggestion
  DocumentUrl: https://glpi-agent.readthedocs.io/
```

File: manifests/g/GLPI-Project/GLPI-Agent/1.11/GLPI-Project.GLPI-Agent.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    configure if GLPI-AgentMonitor should always create a screenshot on new ticket.
+    The default set value is 1 and means to always create a screenshot.
+  * Updated Windows packages 7-Zip commandline tools to v24.08
+  * win32: Updated GLPI-AgentMonitor to v1.4.0
+ReleaseNotesUrl: https://github.com/glpi-project/glpi-agent/blob/HEAD/Changes
+PurchaseUrl: https://glpi-project.org/prices/
+# InstallationNotes:
+Documentations:
+- DocumentLabel: Documentation
+  DocumentUrl: https://glpi-project.org/documentation/
```
Comment (g-bougard) [inline manifests/g/GLPI-Project/GLPI-Agent/1.11/GLPI-Project.GLPI-Agent.locale.en-US.yaml:75]: Documentation URL is wrong as it's the server part one, not agent related one. Better use the RTD documentation url.
```suggestion
  DocumentUrl: https://glpi-agent.readthedocs.io/
```

**Review by g-bougard — CHANGES_REQUESTED** [review]
Hello,

I'm the official glpi-agent maintainer and found few wrong url in your new version submission.

Can you update them ?

**Review by g-bougard — APPROVED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**